Ryujinx.Ui.Common/App/ApplicationLibrary.cs

```diff
@@ -151,7 +105,18 @@ public void LoadApplications(List<string> appDirs, Language desiredTitleLanguage
                         continue;
                     }
 
-                    foreach (string app in GetFilesInDirectory(appDir))
+                    IEnumerable<string> content = Enumerable.Empty<string>();
+
+                    try
+                    {
+                        content = Directory.EnumerateFiles(appDir, "*", SearchOption.AllDirectories);
+                    }
+                    catch (UnauthorizedAccessException)
+                    {
+                        Logger.Warning?.Print(LogClass.Application, $"Failed to get access to directory: \"{appDir}\"");
+                    }
+
+                    foreach (string app in content)
```

Comment:
```suggestion
                    foreach (string app in Directory.EnumerateFiles(appDir, "*", SearchOption.AllDirectories))
```
Github doesn't let me create a comment further down, so I can put everything into the suggestion block.

You should move the foreach-loop into the try block. This allows you to get rid of the `content` variable and you won't need to import `System.Linq`.